src/cpp/src/model_runner.hpp

```diff
@@ -118,24 +134,62 @@ class ModelRunner {
             total_num_tokens += sequence_group->get_num_scheduled_tokens() * num_sequences;
             total_num_blocks += sequence_group->get_num_blocks() * num_sequences;
             max_context_len_val = std::max(max_context_len_val, sequence_group->get_context_len());
+            for (auto seq: sequence_group->get_running_sequences()) {
+                num_generated_ids += seq->get_generated_len();
```

Comment:
probably, we need to compute embeddings only for last generated token, while previously generated tokens can save their embeddings in SequenceGroup.

We can experiment later which approach has better performance

Reply:
Added TODO.